src/Utilities/ErrorHandling/Error.hpp

```diff
@@ -34,48 +35,33 @@
 // 20160415) can't figure out that the else branch and everything
 // after it is unreachable, causing warnings (and possibly suboptimal
 // code generation).
-#define ERROR(m)                                                            \
-  do {                                                                      \
-    disable_floating_point_exceptions();                                    \
-    std::ostringstream avoid_name_collisions_ERROR;                         \
-    /* clang-tidy: macro arg in parentheses */                              \
-    avoid_name_collisions_ERROR << m; /* NOLINT */                          \
```

Comment:
Is this important?

Reply:
No.  The variable was originally called something generic like "ss", and then someone called `ERROR(ss)` and it didn't work.